CoreScriptsRoot/CoreScripts/NotificationScript2.lua

```diff
@@ -175,11 +182,11 @@ PopupText.Text = "This is a popup"
 PopupText.Parent = PopupFrame
 
 --[[ Helper Functions ]]--
-local insertNotifcation = nil
+local insertNotification = nil
```

Comment:
RemoveNotification seems to be commented out of insertNotification so it seems fixable now.